compiler-rt/lib/sanitizer_common/sanitizer_stacktrace_printer.cpp

```diff
@@ -143,7 +143,12 @@ static const char kDefaultFormat[] = "    #%n %p %F %L";
 
 void RenderFrame(InternalScopedString *buffer, const char *format, int frame_no,
                  uptr address, const AddressInfo *info, bool vs_style,
```

Comment:
vs_style is ignored?

Reply:
RenderFrame does use vs_style. Its is not used in symbolizer markup code.

Reply:
why it then forwarded into RenderFrameMarkup?
if they are mutually exclusive, would it be better replace two bool flags with enum?